src/main/resources/application.yml

```diff
@@ -0,0 +1,29 @@
+---
+info:
+  component: Default Single Swagger Document Server for MicroService
+
+spring:
+  profiles: default
+
+server:
+  port: 8014
+
+documentation:
+  swagger:
+    services:
+      -
+        name: Ms Auth
+        url: ${MSAUTH_DOCS}
+        version: 2.0
+      -
+        name: Ms Complaint
+        url: ${MSCOMPLAINT_DOCS}
+        version: 2.0
+      -
+        name: Ms Subsriber
+        url: ${MSSUBSCRIBER_DOCS}
+        version: 2.0
+      -
+        name: Ms Team
+        url: ${MSTEAM_DOCS}
```

Comment:
where is the url ? in heroku args ?

Reply:
url is mapped to config var and config var is kept in heroku
```
        url: ${MSTEAM_DOCS}
in here, ${MSTEAM_DOCS} is config var and it points to msteam docs url
```